JeffShepherd/niche-audience
# Feature/landing view

## What does this PR do?

- Adjust layout and styling of landing view

## How should it be tested in the UI?

- Start application and ensure the basic layout looks good, and the random photo renders
